dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/CommunityRestRepository.java

```diff
@@ -260,4 +270,28 @@ protected void delete(Context context, UUID id) throws AuthorizeException {
             throw new RuntimeException("Unable to delete community because the logo couldn't be deleted", e);
         }
     }
-}
\ No newline at end of file
+
+    /**
+     * Method to install a logo on a Community which doesn't have a logo
+     * Called by request mappings in CommunityRestController
+     * @param context
+     * @param community     The community on which to install the logo
+     * @param uploadfile    The new logo
+     * @return              The created bitstream containing the new logo
+     * @throws IOException
+     * @throws AuthorizeException
+     * @throws SQLException
+     */
+    public Bitstream setLogo(Context context, Community community, MultipartFile uploadfile)
```

Comment:
This should likely return `BitstreamRest`, just like the `CollectionRestRepository.setLogo()` method.